packages/xxd_standalone.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+
+class Xxd_standalone < Package
+  description 'Hexdump utility from vim'
+  homepage 'http://www.vim.org'
+  @_ver = '8.2.2580'
+  version @_ver
+  license 'GPL-2'
+  compatibility 'all'
+  source_url 'https://github.com/vim/vim/archive/v8.2.2580.tar.gz'
```

Comment:
Should this be updated with all the other vim packages to keep the versions consistent?

Reply:
It is the same version as the other vim versions... Do you see different versions?

Reply:
It is the same version as the other vim versions... Do you see different versions?